src/styles/global.scss

```diff
@@ -41,9 +41,7 @@ body {
   }
 }
 
```

Comment:
Maybe it'd be good to add some comment at the top of this file detailing which classes belong in this file which don't.